Gradata/bench/cross_language_scoring.py

```
@@ -0,0 +1,630 @@
+    total_s: float,
+) -> Path:
+    RESULTS_DIR.mkdir(exist_ok=True)
+    tag = datetime.now(UTC).strftime("%Y%m%dT%H%M%S")
+    out = RESULTS_DIR / f"cross-language-scoring-{tag}.md"
+
+    lines: list[str] = []
+    a = lines.append
+
+    a(f"# cross-language-scoring benchmark — {run_date}")
+    a("")
+    a(f"**GRA-1299**: embedding vs BM25 on zero-term-overlap probes  ")
```

Ruff (0.15.13)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)